lib/bolt/inventory.rb

```diff
@@ -93,6 +93,12 @@ def get_targets(targets)
       targets.map { |t| update_target(t) }
     end
 
+    def add_to_group(target, group_name)
+      if group_names.include?(group_name)
```

Comment:
do we need an error raising else here?

Reply:
Should it create a group if the name is unknown?

Reply:
According to ticket description is should error. I forgot about this comment/requirement. I implemented it to raise an error and am working on a test for that now.

Reply:
I think creating a group should be a separate command since we wouldn't know where to put the new group.